spring-data-jpa/src/main/java/org/springframework/data/jpa/repository/query/JSqlParserQueryEnhancer.java

```diff
@@ -342,6 +346,18 @@ private static String detectAlias(PlainSelect selectBody) {
 		return alias == null ? null : alias.getName();
 	}
 
+	/**
+	 * Resolves the alias for the given {@link Merge} statement.
+	 *
+	 * @param mergeStatement must not be {@literal null}.
+	 * @return Might return {@literal null}.
+	 */
+	@Nullable
+	private static String detectAlias(Merge mergeStatement) {
```

Comment:
Is there a reason why you decided to make the method static?

Reply:
I just kept the same behavior for the function just above (static method) to have a homogeneous code :
private static String detectAlias(PlainSelect selectBody)

Reply:
I made a force push ... I hope this is not a mistake